lib/package.js

```diff
@@ -509,6 +509,12 @@
 
   function loadPackageConfigPath(loader, pkgName, pkgConfigPath) {
     var configLoader = loader.pluginLoader || loader;
+    if (configLoader.execute === false) {
+      var config = configLoader.getConfig();
+      configLoader = new configLoader.constructor();
+      configLoader.config(config);
+      configLoader.execute = true;
+    }
```

Comment:
This is exactly what `pluginLoader` is designed for, to take over when execution is disabled. So it's fine.